### PR TITLE
black

### PR DESCRIPTION
**Pull request by almarklein**

### CODE REVIEW

File: tests/test_compute.py [hunk 1]
```diff
@@ -224,9 +229,27 @@ def compute_shader(
...
+        compute_with_buffers(
+            {0: in1},
+            {
+                0: (
+                    -1,
+                    "i",
+                )
+            },
+            compute_shader,
+        )
```
Comment (almarklein) [inline tests/test_compute.py:252]: How is this supposed to be more readable? :(

Comment (Korijn) [reply tests/test_compute.py:252]: Black does not make things more readable... ;) it just enforces rules consistently

Comment (almarklein) [reply tests/test_compute.py:252]: I just wish it would enforce rules that make it more readable ;)

Comment (almarklein) [reply tests/test_compute.py:252]: This is just silly

Comment (almarklein) [reply tests/test_compute.py:252]: They should include a sillyness factor in their algorithm :)

**Pull request merged**